src/components/Checkbox/Checkbox.unit.spec.jsx

```diff
@@ -159,4 +145,74 @@ describe('<Checkbox/>', () => {
     expect(inputs.prop('aria-labelledby')).to.equal('headingId');
     tree.unmount();
   });
+
+  describe('analytics event', function () {
+    it('should NOT be triggered when enableAnalytics is not true', () => {
+      const handleAnalyticsEvent = sinon.spy();
+
+      global.document.body.addEventListener(
+        'component-library-analytics',
+        handleAnalyticsEvent,
+      );
```

Comment:
Ooohhh good call.  It would be used like this, correct?

https://github.com/department-of-veterans-affairs/component-library/blob/32fe46bd5ea27eeb7046abf61f6b0ec4170cf17d/src/components/TextInput/TextInput.unit.spec.jsx#L259-L285

I dropped your function into `test-helpers.js`.

Definitely some room to combine some of these wrappers into a shared object in a `beforeEach` as well.